doc/DeveloperGuide.md

```diff
@@ -94,7 +94,9 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
-
+5. Should be usable by anyone with experience in CLI
+6. Should not be resource heavy
```

Comment:
What is considered a resource? How much resource need to be consumed to be considered as such? Try to be more precise with the requirements.